README.md

```diff
@@ -1 +1,29 @@
-# selt
\ No newline at end of file
+# selt
+selt is a tool used for running Selenium automation on graphical user 
+interfaces on the web.
+
+This tool is a work in progress. The code for selt can be found in run.py. 
+However, this repository also includes an example test project. This will 
+likely be moved into a separate repository at a later time, and selt will be 
+turned into a standalone python package.
+
+Currently, selt only supports the latest version of chrome, chrome-headless, 
+firefox, and firefox-headless.
+
+## Dependencies
+* python
+* selenium
+* termcolor
+
+## Usage
+
+1- cd into your selt directory.
+
+`
```

Comment:
Normally, code "blocks" in a .md are signified with "```", and "`" is reserved for single-lines.

Reply:
Didn't know that!